packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -123,10 +121,82 @@ export class Breadcrumbs implements Integration {
     if (!('document' in global)) {
       return;
     }
+
     // Capture breadcrumbs from any click that is unhandled / bubbled up all the way
     // to the document. Do this before we instrument addEventListener.
     global.document.addEventListener('click', breadcrumbEventHandler('click'), false);
     global.document.addEventListener('keypress', keypressEventHandler(), false);
+
+    // After hooking into document bubbled up click and keypresses events, we also hook into user handled click & keypresses.
+    ['EventTarget', 'Node'].forEach((target: string) => {
+      const proto = (global as any)[target] && (global as any)[target].prototype;
+
+      if (!proto || !proto.hasOwnProperty || !proto.hasOwnProperty('addEventListener')) {
+        return;
+      }
+
+      fill(proto, 'addEventListener', function(
+        original: () => void,
+      ): (
+        eventName: string,
+        fn: EventListenerOrEventListenerObject,
+        options?: boolean | AddEventListenerOptions,
+      ) => void {
+        return function(
+          this: any,
+          eventName: string,
+          fn: EventListenerOrEventListenerObject,
+          options?: boolean | AddEventListenerOptions,
+        ): (eventName: string, fn: EventListenerOrEventListenerObject, capture?: boolean, secure?: boolean) => void {
+          if ((fn as any).handleEvent) {
```

Comment:
It breaks at https://github.com/react-component/drawer/issues/85

Reply:
Fixed in 5.1.1